cedar-policy-cli/src/lib.rs

```diff
@@ -104,6 +104,9 @@ pub enum Commands {
     Visualize(VisualizeArgs),
     /// Create a Cedar project
     New(NewArgs),
+    #[cfg(feature = "partial-eval")]
+    /// Partially evaluate an authorization request
+    PartiallyAuthorize(PartiallyAuthorizeArgs),
```

Comment:
For the experimental validation features I had the CLI print a nice message if you tried to use it without compiling in the feature. This is a little different since it's new subcommand rather than a flag to an existing command, but it'd be nice to the same here if you see an easy way.

Reply:
Good idea. I'll change it.

Reply:
Done.